### PR TITLE
Use `inkscape --shell` to speed-up query processing and PDF rendering

### PR DESCRIPTION
**Pull request by Kobzol**

### CODE REVIEW

File: elsie/inkscape.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            line += character
+            if character == "\n" or line == PROMPT:
+                queue.put(line)
+                line = ""
+
+
+@contextlib.contextmanager
+def svg_file_input(inkscape, svg: str, binary=False):
+    with tempfile.NamedTemporaryFile("wb" if binary else "w", suffix=".svg") as f:
+        f.write(svg)
```
Comment (Kobzol) [inline elsie/inkscape.py:35]: Probably will not work on Windows due to file writer lock.

**Comment by spirali** [comment]
Super nice :fairy:  :tada:

File: elsie/inkscape.py (hunk 1 above)
Comment (spirali) [reply elsie/inkscape.py:35]: We can solved it together with the first windows user :)

Comment (tomc) [reply elsie/inkscape.py:35]: Do you need something tested under windows?  I have inkscape and Py 3.7 installed, wouldn't be too tough to try some examples.

Comment (Kobzol) [reply elsie/inkscape.py:35]: To be honest, we haven't really tested `elsie` under Windows at all. You can try the `master` branch if you can compile some of the examples in the repository root.

**Pull request merged**